client/src/rpc/impls/cfx.rs

```diff
@@ -787,16 +787,19 @@ impl RpcImpl {
     }
 
     fn prepare_block_receipts(
-        &self, block_hash: H256, maybe_pivot_hash: Option<H256>,
+        &self, block_hash: H256, pivot_assumption: H256,
     ) -> RpcResult<Option<Vec<RpcReceipt>>> {
         let exec_info = match self.get_block_execution_info(&block_hash)? {
-            None => return Ok(None),
+            None => return Ok(None), // not executed
             Some(res) => res,
         };
```

Comment:
If there is a pivot chain reorg during processing then we'll probably return `null` because of this part, instead of raising an error. Not sure if it's an issue, and this case seems rare.